src/openms/source/CONCEPT/Constants.cpp

```diff
@@ -154,6 +154,9 @@ namespace OpenMS
 
     namespace UserParam
     {
+      // User parameter name for the M/Z of other chromatograms which have been merged into this one
+      const std::string MERGED_WITH = "merged_chromatogram_mzs";
```

Comment:
is there a more descriptive name?
`MERGED_WITH` is a bit unspecific.
Maybe `MERGED_WITH_MZ_USERPARAM`?

Reply:
or even MERGED_CHROMATOGRAM_MZS

Reply:
done.